Tests/L0/Common-VstsAzureHelpers_/_suite.ts

```diff
@@ -82,6 +82,9 @@ describe('Common-VstsAzureHelpers_ Suite', function () {
         it('(Initialize-AzureSubscription) throws when RM and cert auth', (done) => {
             psr.run(path.join(__dirname, 'Initialize-AzureSubscription.ThrowsWhenRMAndCertAuth.ps1'), done);
         })
+        it('(Initialize-AzureSubscription) throws when invalid environment is passed', (done) => {
```

Comment:
This description is inconsistent with the file name (and appears to be inconsistent with what the test does).